barretenberg/cpp/src/barretenberg/stdlib/primitives/biggroup/biggroup.hpp

```diff
@@ -221,8 +221,8 @@ template <class Builder_, class Fq, class Fr, class NativeGroup> class element {
     element normalize() const
     {
         element result(*this);
-        result.x.assert_is_in_field();
-        result.y.assert_is_in_field();
+        result.x.reduce_mod_target_modulus();
```

Comment:
We need full reduction here as well. This function doesn't really seem to be used anywhere except biggroup tests so this doesn't affect anything anyways.